### ISSUE TITLE
Can't handle newline after function arguments

### ISSUE THREAD

**Issue by fhchl** [KristoforMaynard/SublimeAutoDocstring#45]
Package doesn't work in the following case (nothing happens) that you will often find when using [black](https://github.com/ambv/black)

```python
"""Module doc string.

Blubb.
"""


def func2(
    b
):
    return b
```

It seems like it has to do with the newline after the function arguments.

**Comment by KristoforMaynard** [comment]
This looks like a duplicate of #42, which was fixed in version 0.5.8 (I did a quick copy and paste and your snippet worked for me :/). That being said, I think there are still some edge cases that confuse the parser, so there could be something subtle happening here.

Ultimately, this plugin should transition to using a proper parser like [parso](https://parso.readthedocs.io/en/latest/), but I haven't found the time to look into this too deeply. Anyone that feels like taking a look at the parser is more than welcome :)
